internal/telemetry/service_test.go

```diff
@@ -47,3 +82,20 @@ type testTracker struct {
 func (tracker *testTracker) Track(event event) {
 	tracker.lastTrackedEvent = event
 }
+func testServiceStdoutput(t *testing.T, createFn func() *Service, expected string) {
```

Comment:
`Stdoutput`, while an excellent _portmanteau_, isn't exactly meaningful in this context... let's refactor this to be:

```go
func testServiceOutput(t *testing.T, service *Service, expectedOutput string) {
  t.Helper()

  // rest of code here
}
```